hack/generate/samples/internal/go/memcached-with-webhooks/memcached_with_webhooks.go

```diff
@@ -397,9 +406,15 @@ func (mh *Memcached) implementingController() {
 	// Add imports
 	err := kbutil.InsertCode(controllerPath,
 		"import (",
-		importsFragment)
+		contorollerImportsFragment)
```

Comment:
Nit - this should be:
```suggestion
		controllerImportsFragment)
```

Reply:
Done